pandas/core/resample.py

```diff
@@ -1617,7 +1671,9 @@ def _take_new_index(obj, indexer, new_index, axis=0):
         raise ValueError("'obj' should be either a Series or a DataFrame")
 
 
-def _get_timestamp_range_edges(first, last, offset, closed="left", base=0):
+def _get_timestamp_range_edges(
+    first, last, freq, closed="left", base=0, origin=None, offset=None
```

Comment:
same here and all in this PR. remove the deprecated args, *except* from the user facing constructor. you need to fix the internal code to not use them

Reply:
Should be done in the last commit!